algorithms/CPlusPlus/Linked-Lists/singly.cpp

```diff
@@ -36,7 +39,7 @@ class SinglyLinkedList {
             // change head point to new node
             this->head = temp;
         }
-    
+
         void insertAtEnd(T data){
 
             Node<T> *newNode = new Node<T>(data);
```

Comment:
Alright thanks, I'll see what I can do to fix this

Reply:
according to me the best way to do this would be to either perform garbage collection inside the list's destructor or make use of smart pointers to do that for you.